### PR TITLE
Add initial support for DE version

### PR DESCRIPTION
**Pull request by arisinfenix**
## Description
Add option to display DE version as requested in #1052.

## Features
Initially supports only Plasma.

### CODE REVIEW

**Comment by arisinfenix** [comment]
## Update
Added more DEs, now supports:

Plasma
MATE
Xfce
GNOME
Cinnamon
Deepin
Budgie

File: neofetch [hunk 1]
```diff
@@ -1585,20 +1596,36 @@ get_de() {
...
-            de=$(cinnamon --version)
-            de=${de:-Cinnamon}
-        ;;
-
-        *GNOME*)
-            de=$(gnome-shell --version)
-            de=${de/Shell }
-        ;;
+        *GNOME*) de=GNOME ;;
+        *MUFFIN*|Cinnamon) de=Cinnamon ;;
```
Comment (dylanaraps) [inline neofetch]: This could now just be: `*MUFFIN*) de=Cinnamon ;;` as `de` will equal Cinnamon anyway.

File: neofetch [hunk 2]
```diff
@@ -1585,20 +1596,36 @@ get_de() {
...
+            "MATE"*)     de_ver=$(mate-session --version) ;;
+            "Xfce"*)     de_ver=$(xfce4-session --version) ;;
+            "GNOME"*)    de_ver=$(gnome-shell --version) ;;
+            "Cinnamon"*) de_ver=$(cinnamon --version) ;;
+            "Deepin"*)   de_ver=$(awk -F'=' '/Version/ {print $2}' /etc/deepin-version) ;;
+            "Budgie"*)   de_ver=$(budgie-desktop --version) ;;
+            "LXQt"*)     de_ver=$(lxqt-session --version) ;;
+            "Lumina"*)   de_ver=$(lumina-desktop --version 2>&1) ;;
+            "Trinity"*)  de_ver=$(tde-config --version) ;;
+        esac
```
Comment (dylanaraps) [inline neofetch]: Add a newline here.

File: neofetch [hunk 3]
```diff
@@ -2752,6 +2779,9 @@ get_style() {
         # Get DE if user has disabled the function.
         ((de_run != 1)) && get_de
 
+        # remove version from $de
```
Comment (dylanaraps) [inline neofetch]: `# Remove version from '$de'.`

**Comment by dylanaraps** [comment]
This looks really good, left some comments. :+1:

**Comment by arisinfenix** [comment]
Thanks, applied and pushed the changes.

**Comment by dylanaraps** [comment]
Let me know when this is ready for merging. :+1:

**Comment by arisinfenix** [comment]
For now I'm not aware of any other DE to add. Just updated the man page, so I think it's ready now.

**Comment by dylanaraps** [comment]
Thanks :+1:

**Pull request merged**